### ISSUE TITLE
Clear cache needed to pick up tailwind changes

### ISSUE THREAD

**Issue by kimberleyamackenzie** [pulumi/docs#11685]
### What happened?

We shipped some code changes that were all styling/Tailwind, including a Theme PR with a new tailwind config breakpoint (shipped several days prior to the code changes) and some Tailwind class changes in pulumi-hugo.  The parts of those changes that relied on the new breakpoint were missing for some folks, until they cleared their cache.

### Steps to reproduce

We do not have a reliable repro, since after a cache clear, all happened as expected.  I am unable to repro this on prod now using new browsers/windows.  Part of solving this bug will likely be finding a repro to ensure we know the exact cause.

### Expected Behavior

Comment from @cnunciato: 

> clearing the browser cache seemed to help — but i’m wondering why that helped, given the way we name our CSS and JS bundles (which is set up to bust browser cache through naming). we haven’t seen this to my knowledge before (at least not since moving to the bucket-per-commit model).
> 
> https://github.com/pulumi/docs/blob/b6a72618ff66830bac8000e5e1a2c75614b7e1eb/scripts/build-site.sh#L11-L20
> .. where build_identifier is based on the Git SHA:
> https://github.com/pulumi/docs/blob/b6a72618ff66830bac8000e5e1a2c75614b7e1eb/scripts/common.sh#L82

### Actual Behavior

You should be able to pick up the changes without needing to clear your cache.

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

**Comment by jkodroff** [comment]
@claude That's not right - this repo was originally called pulumi-hugo. Run your analysis again with that fact in mind.

**Comment by jkodroff** [comment]
Cam says Jeff fixed this. Closing.
